R/knitr-engine.R

```diff
@@ -82,6 +79,9 @@ eng_python <- function(options) {
       knitr::include_graphics(path)
   }
   
+  # set up figure dimensions
+  plt$rc("figure", figsize = tuple(options$fig.width, options$fig.height))
```

Comment:
If it's useful or better to have figure height in pixels you might want `out.width.px` (that's what we use in htmlwidgets: https://github.com/ramnathv/htmlwidgets/blob/346f87c390d118a8a2c2e13c2faa0b3a5cf9e5d6/R/sizing.R#L210-L211)

Reply:
From what I can see thus far, most of the `matplotlib` APIs accept inches + DPI rather than raw pixels, hence why I'm using these directly.